applications/DropletDynamicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -0,0 +1,42 @@
+//    |  /           |
```

Comment:
Same comment. If not needed yet I wouldn't add it. You can add it in the future as required.

Reply:
There would be an electromagnetic solver maybe as a utility. I am not sure though (might be a process).